PMM-7 Update mysql_prerequisites.md (#1260)
percona/pmm-doc: docs/get-started/backup/mysql_prerequisites.md

   !!! caution alert alert-warning "Important"
    If PMM Server runs as a Docker container, enable backup features at container creation time by adding `-e ENABLE_BACKUP_MANAGEMENT=1` to your `docker run` command.

> 2. Check that the [PMM Client](../../setting-up/client/index.md) is installed and running on the node where backup should be done.

Reviewer: ```suggestion
2. Check that the [PMM Client](../../setting-up/client/index.md) is installed and running on the node where the backup will be performed.
```

Author: Thank you

    - [`qpress`][PERCONA_QPRESS].

!!! caution alert alert-warning "Important"

>        The versions of each must be compatible with the installed version of MySQL.

Reviewer: ```suggestion
       Make sure that the versions of xtrabackup, xbcloud, xbstream, and qpress are fully compatible with the currently installed version of MySQL on the system. 
```
